docs/how-to-build-with-ddn/with-mysql.mdx

```diff
@@ -0,0 +1,435 @@
+        "title": "Hello World",
+        "content": "Charlie joins the conversation."
+      }
+    ]
+  }
+}
+```
+
+### Step 14. Create a relationship
+
+```sh title="Since there's already a foreign key on the posts table in MySQL, we can easily add the relationship:"
+ddn relationship add my_mysql "mydb.posts"
+```
+
+You'll see a new metadata object added to the `app/metadata/MydbPosts.hml` file of kind `Relationship` explaining the
+relationship between `posts` and `users`.
+
+### Step 15. Rebuild your project
+
+```sh title="As your metadata has changed, create a new build:"
+ddn supergraph build local
+```
+
+```sh title="Bring down the servies by pressing CTRL+C and start them back up:"
+ddn run docker-start
+```
+
+### Step 16. Query using your relationship
+
+```graphql title="Now, execute a nested query using your relationship:"
+query GetPosts {
+  mydbPosts {
+    id
+    title
+    content
+    mydbUser {
+      id
+      name
+      age
+    }
+  }
+}
+```
+
+```json title="Which should return a result like this:"
+{
+  "data": {
+    "mydbPosts": [
+      {
+        "id": 1,
+        "title": "My First Post",
+        "content": "This is Alice's first post.",
+        "user": {
+          "id": 1,
+          "name": "Alice",
+          "age": 25
+        }
+      },
+      {
+        "id": 2,
+        "title": "Another Post",
+        "content": "Alice writes again!",
+        "user": {
+          "id": 1,
+          "name": "Alice",
+          "age": 25
+        }
+      },
+      {
+        "id": 3,
+        "title": "Bob's Post",
+        "content": "Bob shares his thoughts.",
+        "user": {
+          "id": 2,
+          "name": "Bob",
+          "age": 30
+        }
+      },
+      {
+        "id": 4,
+        "title": "Hello World",
+        "content": "Charlie joins the conversation.",
+        "user": {
+          "id": 3,
+          "name": "Charlie",
+          "age": 35
+        }
+      }
+    ]
+  }
+}
+```
+
+## Next steps
+
+Congratulations on completing your first Hasura DDN project with MySQL! 🎉
+
+Here's what you just accomplished:
+
+- You started with a fresh project and connected it to a local MySQL database.
+- You set up metadata to represent your tables and relationships, which acts as the blueprint for your API.
+- Then, you created a build — essentially compiling everything into a ready-to-use API — and successfully ran your first
+  GraphQL queries to fetch data.
+- Along the way, you learned how to iterate on your schema and refresh your metadata to reflect changes.
+- Finally, we looked at how to enable mutations and insert data using your new API.
+
+Now, you're equipped to connect and expose your data, empowering you to iterate and scale with confidence. Great work!
+
+Take a look at our MySQL docs to learn more about how to use Hasura DDN with MySQL. Or, if you're ready, get started
+with adding [permissions](/reference/metadata-reference/permissions.mdx) to control access to your API.
```

Comment:
Probably better to link to /auth/permissions/index.mdx instead of the metadata reference page here? I think it the same on the other how to build pages too which we'd need to change.

Reply:
Yep, I agree. Let's not make this PR dependent on that. I've created [this chore ticket](https://linear.app/hasura/team/DOC/cycle/active?filter=eyJhbmQiOlt7ImFzc2lnbmVlIjp7Im9yIjpbeyJpZCI6eyJpbiI6WyJlNWUzNDFjZC0yZGIzLTQ1NzQtODk5NS02NTExYzcxYzM4MzAiXX19XX19XX0&layout=board&ordering=manual&grouping=workflowState&subGrouping=none&showCompletedIssues=all&showSubIssues=true&showTriageIssues=false) and will grab it later today or tomorrow 👍